cli/src/main/java/ai/wanaku/cli/main/support/CollectionHelper.java

```diff
@@ -0,0 +1,48 @@
+package ai.wanaku.cli.main.support;
+
+import java.util.Collection;
+import java.util.Map;
+
+public class CollectionHelper {
```

Comment:
We could merge this one with `core/core-util/src/main/java/ai/wanaku/core/util/CollectionsHelper.java`.